common/src/tflite.h

```diff
@@ -29,6 +29,9 @@
 // Sets up TfLite with a given model
 void tflite_load_model(const unsigned char* model_data, unsigned int model_length);
 void tflite_set_input_zeros();
+void tflite_load_model(const unsigned char* model_data,
```

Comment:
Seems like a duplicate definition is being introduced here? It's harmless, since it matches the one on line 30 above, but I guess it would be good to tidy it up.

Reply:
I think it's just a reformat of the same line, due to length.
And line 31 has added the explicit "void".